MainWindow.xaml.cs

```diff
@@ -267,7 +306,7 @@ private void BrowseFolder()
                     {
                         FileName = dicPath,
                         Arguments = parameters,
-                    },
+                    }
```

Comment:
This is a pedantic nit: I leave the dangling commas so that future changes touch less code. It's something I've had to do for work so it's built in. If you don't mind returning that, I'd appreciate it.

Reply:
Sure I will but I don't guarantee to remember to put them where needed since it's not in my code taste.